org.upscayl.app.yml

```diff
@@ -0,0 +1,70 @@
+id: org.upscayl.app
```

Comment:
The domain needs to exist. If you don't own it, please change ID to `io.github.upscayl.Upscayl`.

Reply:
> The domain needs to exist. If you don't own it, please change ID to `io.github.upscayl.Upscayl`.

@barthalion Do I have to change just this field or all of them?